src/core/Core_result.mli

```diff
@@ -8,9 +8,16 @@ type 'a match_result = {
 }
 [@@deriving show]
 
+(* See the .ml file for why we have this instead of just matches. *)
+type processed_match = {
+  pm : Pattern_match.t;
+  is_ignored : bool; [@default false]
```

Comment:
what are those @default for?

Reply:
I'm not sure. I just saw them around but didn't really know what they ddi, so I just added them. I'll just remove them, I guess.